kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -1,6 +1,205 @@
-define([], () => {
+define([
+    'common/html',
+    'common/format',
+    'common/ui',
+], (
+    html,
+    format,
+    UI,
+) => {
     'use strict';
 
+    const t = html.tag,
+        span = t('span'),
+        cssBaseClass = 'kb-job-state';
+
+    /**
```

Comment:
Heads up! New functions in this file. They will be used by code coming up in another PR in the near future.